packages/babel-helpers/src/helpers.js

```diff
@@ -814,3 +814,125 @@ helpers.applyDecoratedDescriptor = defineHelper(`
         return desc;
     }
 `);
+
+helpers.wrapRegExp = defineHelper(`
+  var kGroups = Symbol("@@babel/groups");
+
+  function BabelRegExp(re, groups) {
```

Comment:
could this be wrapped in an iife? all those assignments at the moment will prevent tree-shaking if `BabelRegExp` class stays unused. Im not sure if we can annotate helper as `#__PURE__` at the moment though (I think comments in the helpers are ignored)

Reply:
we could probably also use a redefining helper technique here to contain this whole declaration within a single top level node

it seems though that it would actually be cleaner if we could transpile helpers with user settings - this could be written as an actual class